stdlib/2/inspect.pyi

```diff
@@ -61,6 +85,12 @@ ArgInfo = NamedTuple('ArgInfo', [('args', List[str]),
                                  ('locals', Dict[str, Any]),
                                  ])
 
+Arguments = NamedTuple('Arguments', [('args', List[Union[str, list]]),
```

Comment:
Missing type arguments to the inner `list`.

Reply:
The description in the code was "'args' is a list of argument names (possibly containing nested lists)", which I think means that the type is List[Union[str, List[Union[str, ...]]]].  Is there a way to type that precisely?

Reply:
No, but in that case I'd prefer explicit `List[Any]` over `list`.